daml-foundations/daml-ghc/tests/Map.daml

```diff
@@ -59,18 +55,6 @@ testDelete = scenario do
   [(2, False), (3, True), (4, False), (5, False)] === toList (delete 1 (fromList [(3, True), (1, False), (4, False), (2, True), (5, False), (2, False), (1, True)]))
   [(1, False), (2, True)] === toList (delete 3 (fromList [(2, False), (1, True), (2, True), (1, False)]))
 
-testMerge = scenario do
```

Comment:
Can we not get this back? Seems a shame.

Reply:
I'll add it back in a separate PR. We'll need `merge` in `DA.TextMap` for that purpose. See https://github.com/digital-asset/daml/issues/881.